pkgs/os-specific/linux/rdma-core/default.nix

```diff
@@ -3,7 +3,7 @@
 } :
 
 let
-  version = "34.0";
+  version = "35.0";
 
 in stdenv.mkDerivation {
   pname = "rdma-core";
```

Comment:
Please convert that let in into a rec and target master.

Reply:
I think I have asked that question before: what is the motivation behind converting a perfectly valid `let` into a `rec`?

Reply:
Cleanup

Reply:
I don't see how this contributes to cleanup as it does not change the readability of the derivation. In fact one could argue that the use of `rec` is an anti-pattern: https://nix.dev/anti-patterns. If it does anything, requesting such changes creates unnecessary work.

Reply:
If you take at the examples on the website they are of a different nature and not really comparable to this case. src is a fixed output derivation and will not change when you just change version. Also this is the default template for new packages.

I happy to remind people to modernize their expressions even if it is extra work but if you resist.

Reply:
We should really leave these small, largely stylistic differences up to the maintainer. It's not worth tiring out maintainers and committers (and yourself) with endless discussions about them.

Reply:
@danieldk 

Agreed, or at least we adopt a formatter which is opinionated on the matter. Unless something is actively detrimental, I don't think we need to contest two ways to arrive the same destination.